Code-the-Dream-School/cc-prac-team4-front
# PP 130 connect pet adoption application page with backend

Pet Adoption Application page was connected with backend.
src/component/Adoption/ConfirmApplication.js

Some improvement on Donation page:
src/component/Donate/Donate.module.css

Some UI improvement:
src/component/Home/Home.module.css

Changed px to rem:
src/component/NoFavorites/NoFavorites.module.css

Removed background picture from Contact page for mobile version:
src/component/layout/Contact/Contact.module.css

Ask us about local var .env!
**Testing Steps:**
Choose the pet from Favorites Page or Pet Details page
Press Adopt button, User will be navigate to Confirm Application page
Fill the Form correctly (zip code, phone and email fields have validation)
Check Agreement to terms and conditions
Press Submit button
If no errors User will see pop modal with message from backend and will receive letter with copy of form.
Also letter with Application form will receive  by shelter staff (according by SHELTER_EMAIL).
After closing pop Modal User will be navigating to Home page.

Pop up message to user after submitting form:
<img width="1440" alt="Screen Shot 2023-07-06 at 3 58 25 AM" src="https://github.com/Code-the-Dream-School/cc-prac-team4-front/assets/112579936/5adfb947-7dfb-429c-972d-371b54a2db3c">

Template of Adoption Application form that will be sent on email  of the shelter and the user:
<img width="1440" alt="Screen Shot 2023-07-06 at 4 35 01 AM" src="https://github.com/Code-the-Dream-School/cc-prac-team4-front/assets/112579936/04f02e9a-86a7-4177-a667-104e36e17e9d">
